test/e2e/e2e.test.ts

```diff
@@ -420,8 +444,39 @@ describe('keyword preservation', () => {
       config,
     });
 
+    //Dumping without keyword preservation so we can assert that remote values will overwrite local values
+    await dump({
+      output_folder: workDirectory,
+      format: 'directory',
+      config: {
+        ...config,
+        AUTH0_PRESERVE_KEYWORDS: false,
+      },
+    });
+
+    const jsonWithoutPreservation = JSON.parse(
+      fs.readFileSync(path.join(workDirectory, 'tenant.json')).toString()
+    );
+
+    expect(jsonWithoutPreservation.friendly_name).to.equal('This tenant name should be preserved');
+    expect(jsonWithoutPreservation.enabled_locales).to.deep.equal(['en', 'es']);
+    expect(jsonWithoutPreservation.support_email).to.equal('support@travel0.com');
+    expect(jsonWithoutPreservation.support_url).to.equal('https://travel0.com/support');
+
+    const emailTemplateJsonWithoutPreservation = JSON.parse(
+      fs.readFileSync(path.join(workDirectory, 'emails', 'welcome_email.json')).toString()
+    );
+
+    expect(emailTemplateJsonWithoutPreservation.resultUrl).to.equal('https://travel0.com/welcome');
+
+    expect(
+      fs.readFileSync(path.join(workDirectory, 'emails', 'welcome_email.html')).toString()
+    ).to.contain('This tenant name should be preserved');
+
+    emptyDirSync(workDirectory);
```

Comment:
These new E2E assertions are to ensure that keyword replacement worked when keyword preservation disabled. Otherwise, false positives could occur during testing.